include/msvc/libintvector.h

```diff
@@ -525,11 +554,10 @@ typedef uint32x4_t Lib_IntVector_Intrinsics_vec128;
 #define Lib_IntVector_Intrinsics_vec128_rotate_right_lanes64(x0, x1)	\
   (vextq_u64(x0,x0,x1))
 
-
-/*
 #define Lib_IntVector_Intrinsics_vec128_shuffle32(x0, x1, x2, x3, x4)	\
-  (_mm_shuffle_epi32(x0, _MM_SHUFFLE(x1,x2,x3,x4)))
+  ((uint32x4_t){((uint32x4_t)x0)[x1],((uint32x4_t)x0)[x2],((uint32x4_t)x0)[x3],((uint32x4_t)x0)[x4]})
```

Comment:
why bother casting x0 hre to uint32x4_t since vec128 is already defined to be that type?

Reply:
Updated. Thanks!